main.js

```diff
@@ -0,0 +1,87 @@
+// Dan Piston Eval 1 - Towers of Hanoi - 3/25/2021
+
+// Any time you iterate through an array, you must refrain from using for or while loops - instead you must use the Array helper methods except for forEach - you may not use forEach.
+
+// array object responsible for maintaining state of board
+var board = [
+  [5, 4, 3, 2, 1],
+  [],
+  [],
+];
+
+var numberOfDiscs = board[0].length; // for future use
+
+// print board using MAP at least once 
+var printBoard = function (x) {
+  var printedBoard = x.map(function (innerArray) {
+    return '---' + innerArray.join(' ') 
+  });
+  console.log(printedBoard.join('\n'));
+}
+
+printBoard(board);
+
+// moveDesc to play the game as board.moveDesc(1, 3) for example
+board.moveDisc = function (oldPegNumber, newPegNumber) {
```

Comment:
This is a bad practice - you're adding a `moveDisc` prop to an array.

It'd be better do something like:

```js
var board = {
  pegs: [...],
  moveDisc: function...
}